x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types/bedrock.ts

```diff
@@ -435,7 +435,6 @@ export default function bedrockTest({ getService }: FtrProviderContext) {
                   message: 'Hello world',
                   isEnabledKnowledgeBase: false,
                   isEnabledRAGAlerts: false,
-                  llmType: 'bedrock',
```

Comment:
We should decouple this test from our GenAI code so we aren't triggering a ResponseOps review for changes to the internal execute API. 

The reason this test is here is because I wanted to add a test for the Bedrock subaction `invokeStream`. However, the integration tests here use the execute route, which does not support streaming. In order to call this subaction, one must implement a own route to call the actions api directly. The only places that exist are within solutions code. That is why the test calls the security GenAI execute API instead of the actions execute API. 

It would be nice if the actions team could extend the execute route to support streaming. That way, we can update this test to call the actions execute API and ResponseOps will not have to review every change to our GenAI route. @ymao1 I cannot remember if there is an existing issue to add support for streaming to execute route. Is there plans for that? Maybe we could skip this test until that is implemented?

Reply:
I don't believe we have an existing issue for this. Feel free to create one and we will prioritize as capacity allows or PR contributions are welcome as well!